packages/mongodb-downloader/README.md

```diff
@@ -0,0 +1,39 @@
+## mongodb-downloader
+
+A simple library to download MongoDB binaries for different platforms and versions.
+
+### Migrating from v0.4 to v0.5
```

Comment:
Seems like it got bumped to v0.5 since I last wrote this so will update this to be 0.5->0.6